packages/catalog-model/src/schema/EntityMeta.schema.json

```diff
@@ -53,6 +53,12 @@
       "examples": ["default", "admin"],
       "minLength": 1
     },
+    "title": {
+      "type": "string",
+      "description": "A nice display name of the entity, to be presented in user interfaces instead of the name property, when available.",
```

Comment:
```suggestion
      "description": "A display name of the entity, to be presented in user interfaces instead of the name property, when available.",
```

Reply:
hehe, addressed these